src/Transitions/Collapse.js

```diff
@@ -0,0 +1,94 @@
+/*eslint-disable react/prop-types */
+'use strict';
+import React from 'react';
+import Transition from './Transition';
+import domUtils from '../utils/domUtils';
+import createChainedFunction from '../utils/createChainedFunction';
+
+let capitalize = str => str[0].toUpperCase() + str.substr(1);
+
+// reading a dimension prop will cause the browser to recalculate,
+// which will let our animations work
+let triggerBrowserReflow = node => node.offsetHeight; //eslint-disable-line no-unused-expressions
```

Comment:
Pff.. no :) eslint throws warning in that case :smile: 
then `return` + explaining comment
```js
// reading a dimension prop will cause the browser to recalculate,
// which will let our animations work
function triggerBrowserReflow(node) { return node.offsetHeight; }
```